### ISSUE TITLE
[Design/Research] My VA: Plan discovery for how can we better elevate action items

### ISSUE THREAD

**Issue by ajakabcin** [department-of-veterans-affairs/va.gov-team#24967]
## Background
As part of long-term research and continued improvements to My VA post-launch we need to do some discovery/research prep around how to better elevate action items in the dashboard.

## Tasks
- [x] Create a discovery plan
- [x] Audit of action items on va.gov
- [x] Conduct comparative analysis of how action items are surfaced on other websites/tools
- [x] Determine additional deliverables and tickets are needed to execute discovery plan

**Comment by Samara-Strauss** [comment]
@ajakabcin @tressaellen I don't think I did a good job of communicating what I thought needed to happen for this work in this sprint. Here's what I think needs to happen:

- General planning.
- Most likely, a comparative analysis of how action items are surfaced on other websites/tools. There may already be some good documentation on this from Liz's initial My VA discovery. If you don't feel this is necessary, let me know. I do think it would be helpful, though, to do this before getting into any sketching or low-fi mockups.
- Low-fi sketching/mockups: Iterate on a few ideas, keeping in mind how or if this might change the current design, and in what ways. I see you already have this documented in #24970.

I'm hoping this helps with refining what you expect to happen in this ticket and in general this sprint. Let me know if you have questions. Happy to discuss!

**Comment by tressaellen** [comment]
@Samara-Strauss - I had asked AJ to purposefully make this ticket vague so I could spend some time thinking through what needed to happen and then add them it to the ticket (Sorry, I probably should've communicated that!). 

I do think everything you mentioned needs to happen - so we can just track what tasks are outlined here for now and if additional tasks come up as I'm working through things then I'll add them. 

Once I'm through creating a discovery plan I think I'll have a better idea of exactly what should be in this ticket.

**Comment by tressaellen** [comment]
Hello! I've put together [this document](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/action-items-discovery/action-items-discovery-questions.md) to help capture my thoughts/questions for this work.  

In thinking through next steps for this, the only additional item that I think should be included in this ticket and work is a quick audit of va.gov to document how we currently handle notifications/action items on different parts of the site. This will help us determine if we are successfully elevating action items currently anywhere on va.gov and if so, if there is any part of that experience that we could duplicate.  

If this is documented, this will also minimize duplicate work as there are a couple other teams exploring notifications/action items (maybe one of them has already done this audit? - I'll ask around)

**Comment by Samara-Strauss** [comment]
> In thinking through next steps for this, the only additional item that I think should be included in this ticket and work is a quick audit of va.gov to document how we currently handle notifications/action items on different parts of the site. This will help us determine if we are successfully elevating action items currently anywhere on va.gov and if so, if there is any part of that experience that we could duplicate.

Sounds good! I don't *think* we have action item alerts beyond the in-line alert (eg. "Are you sure?" or the address validation alert, etc), which isn't the same as what we're talking about here — effectively updates that require a user to take action on a thing, like a new message from their doctor or an application that's about to expire. But, it's always worth checking out. I don't know every product on VA.gov in and out, so can't hurt to ask around!

> If this is documented, this will also minimize duplicate work as there are a couple other teams exploring notifications/action items (maybe one of them has already done this audit? - I'll ask around)

What other teams?! We should be coordinating with them.

**Comment by tressaellen** [comment]
> What other teams?! We should be coordinating with them.

The debt resolution team and the healthcare team are just starting to explore notifications, and I know VAOS started work on it as well and then had to pivot. I have a meeting next week with the debt team and healthcare team to collaborate!

**Comment by tressaellen** [comment]
> Sounds good! I don't think we have action item alerts beyond the in-line alert (eg. "Are you sure?" or the address validation alert, etc), which isn't the same as what we're talking about here — effectively updates that require a user to take action on a thing, like a new message from their doctor or an application that's about to expire. But, it's always worth checking out. I don't know every product on VA.gov in and out, so can't hurt to ask around!

And sounds good - I'll start poking around a bit and just document if I find anything! I don't think it'll be super time consuming or anything.

**Comment by Samara-Strauss** [comment]
> The debt resolution team and the healthcare team are just starting to explore notifications, and I know VAOS started work on it as well and then had to pivot. I have a meeting next week with the debt resolution designers and healthcare designer to collaborate!

Thank you for alerting me to this. I had no idea. I will check in with their PMs to make sure this work isn't too duplicative. We don't need multiple notification styles within VA.gov, and our team should really be leading this kind of work.

> And sounds good - I'll start poking around a bit and just document if I find anything! I don't think it'll be super time consuming or anything.

Sounds good, let me know what you find!

**Comment by Samara-Strauss** [comment]
OK, I talked to each PM and it sounds like the health care work is about implementing email/text (so not on-site action item alerts/notifications like we're talking about), and I know the debt team wants to work with us when we're ready. Still, please keep me posted on the meetings with their designers so we can make sure there's not duplicate work happening.

**Comment by tressaellen** [comment]
Okay will do! Thank you for checking! I'll keep you posted.

**Comment by tressaellen** [comment]
Hello! I just wanted to post an update in here since I will be out for the rest of the sprint. Here is the [VA audit](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/action-items-discovery/va.gov-action%20item-audit.md) - this will likely be a living document since I'm waiting on some other teams to get back to me. The most important takeaway from that is that **we are not currently doing anything exactly what we're planning for My VA.** There are some aspects from some other projects that might be useful to keep in mind as we move forward.

I'm in the process of writing the [document for the comparison analysis](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/action-items-discovery/action-items-comparative%20analysis.md) - **this is a work in progress and I will finish this when I get back on Wednesday.** In the takeaway section I've just started writing notes - this will be much more robust when I get back. I just wanted to give an idea of where I'm at here.

**Comment by tressaellen** [comment]
@Samara-Strauss the [comparative analysis](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/action-items-discovery/action-items-comparative%20analysis.md) is all wrapped up - I might add some small things here and there as I keep working through things. I already have some ideas about where this could go, so I'll start sketching and working through things. 

Let me know if you have any thoughts or questions!

**Comment by Samara-Strauss** [comment]
Just getting a chance to review this now! Sorry for the delay!

> Status changes
> Things the company/site wanted the user to know

I would not consider these action items, though of course they are part of the broader category of notifications. I think it's ok to mention them, but I would make that differentiation. The goal of the action item work is to focus first on things that require the veteran to actually do something, like:

- If something is due (time-sensitive) (you already mention)
- Missing information (assuming that action is needed to provide this information)
- Some sort of response is needed (like reading an unread message, though this overlaps with "missing information" in some ways)

For status changes or things a company wants you to know, action is generally not required. The reason I am making this distinction is to help scope a likely MVP more closely, so we're not focused on all the notifications all the time right away, and we can think more about how to scale on-sight notifications strategically.

> In the sites observed the type of action item that is always prioritized is any that are time sensitive.

This is great insight! Even within action items, there is priority.

> FAFSA is a much less personalized approach to action items. They have all use cases broken out into check lists, a user could select which one they are and see a list of links to help them make sure they have everything done. This seems fairly vague, and not that helpful when it comes to action items that are time-sensitive. However, this might be an interesting way to approach a first-time user experience.

Agreed — I think the checklist approach is great for set processes, but probably doesn't apply to one-off action items.

> The landing page/dashboard of My Chart is just a running list of action items/reminders all in one place. This seems like a very simple and effective approach, and also seems like something that could work really well on My VA. 

Very close to the My VA-as-a-newsfeed approach we originally explored. Very interesting!

----

Great work! I didn't call out all the valuable insights you collected; there were many of them! I think it's safe to wrap this work and move on to explorations. You've got a lot to work with, and I'm excited to see what you come up with! 

One note on that — please do focus solely on action items in your explorations. If a notification/update doesn't require action, then it doesn't need to be included. For example, on the current My VA, we highlight the most recently updated claim, but this doesn't require action from a user (with the exception of the needing to upload evidence use case, but we don't have a way to elevate that yet based on how the code works). However, reading 2 unread messages IS an action item because it requires the user review those messages to determine whether further correspondence is needed. Other action items that we don't account for on My VA but are currently being worked on by other teams are (1) needing to fill out forms before an appointment and (2) making payments to the VA if someone has a debt (which is unfortunately a thing). So there's a lot to work with in your explorations! Let me know if you have any questions about this differentiation.

**Comment by tressaellen** [comment]
Thank you for this!

> please do focus solely on action items in your explorations. If a notification/update doesn't require action, then it doesn't need to be included.

This is great  - I'll be sure to do this. I'm planning on putting together a spreadsheet with all of the potential action items/notification/updates so I can get a better idea of how to prioritize things - I'll likely be reaching out early next week to go over that, just to make sure I'm on the right track.

If there's nothing else, then I think we're good to close this ticket.
